### DISCUSSION TITLE
Does Hydrogen send MIDI beat clock messages over its MIDI output?

### DISCUSSION THREAD

**Discussion by SuperSonicHub1** [hydrogen-music/hydrogen#1910] [General]
If so, how do I enable it. The MIDI device I'm using is `Hydrogen:Hydrogen Midi-Out`.

**Comment by theGreatWhiteShark** [comment]
Hey @SuperSonicHub1 ,

No. Not yet. But receiving and interpreting MIDI beat clock messages is on the [shortlist](#1644) of new features for version 1.3 Adding the capability to send those signals too would make the whole feature more consistent (and easier to test).

**Comment by theGreatWhiteShark** [comment]
Implemented in #2187
